src/js/components/Table/Paginated/PaginationButtons.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button } from "../../Button";
+import { ButtonGroup } from "../../ButtonGroup";
+import { getLastPageIndex } from "./helpers";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+
+const propTypes = {
+  totalItems: PropTypes.number.isRequired,
+  itemsPerPage: PropTypes.number.isRequired,
+  currentPage: PropTypes.number.isRequired,
+  setCurrentPage: PropTypes.func.isRequired,
+  style: PropTypes.object
+};
+
+const PaginationButtons: React.FunctionComponent<PropTypes.InferProps<
+  typeof propTypes
+>> = ({ totalItems, itemsPerPage, currentPage, setCurrentPage, style }) => {
+  const lastPageIndex = getLastPageIndex(totalItems, itemsPerPage);
+  const buttonLength = totalItems >= 0 ? lastPageIndex : 0;
+  const MAX_MIDDLE_BUTTONS = 3;
+  const buttonMapper = Array(
+    buttonLength > MAX_MIDDLE_BUTTONS ? MAX_MIDDLE_BUTTONS : buttonLength
+  ).fill(currentPage - 1);
+
+  return (
+    <ButtonGroup className="ButtonGroup" style={style}>
+      <Button
+        className="Button"
+        isDisabled={currentPage === 1}
+        onClick={() => setCurrentPage(currentPage - 1)}
+        type="button"
+      >
+        Previous
+      </Button>
+      <Button
+        className="Button"
+        isDisabled={1 === currentPage}
+        onClick={() => setCurrentPage(1)}
+        type="button"
+      >
+        1
+      </Button>
+      {buttonLength > MAX_MIDDLE_BUTTONS && currentPage > MAX_MIDDLE_BUTTONS ? (
+        <Button className="Button" type="button">
+          ...
+        </Button>
```

Comment:
This may be more of a design question. Wouldn't we expect something to happen when we click this? I would thinking clicking this would display the hidden page numbers so that you can navigate to a specific page.

Reply:
I think the main point is to remain consistent with the button styling, but indicate that there are more pages than can be seen currently. We can definitely circle back and update this if there should be an action the user can take with these buttons.

Reply:
I think I'm misunderstanding. Why would we want something styled like a button if it doesn't do anything when you click it?